test/tests.js

```diff
@@ -1832,6 +1833,8 @@ user.getUserIdentities is not a function,\n`;
         var purchaseEventProperties = window.braze.purchaseEventProperties[0];
 
         purchaseEventProperties.should.eql(expectedPurchaseEvent);
+
+        reportService.event.should.be.ok();
```

Comment:
I feel like this should be more like
`        reportService.event.should.have.property(
            'EventName',
            'eCommerce - Purchase'
        );`

Reply:
I think being more specific is a good call.  Will update.